### PR TITLE
Add meshery broker description

### PR DESCRIPTION
**Pull request by pewpewnor**
**Description**

This PR adds meshery broker description from https://docs.meshery.io/concepts/architecture/broker to resolve issue  #509 

**Notes for Reviewers**

Please let me know if the description needs to be edited or anything else. Thank you.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,11 @@ MeshSync (Service Mesh Synchronization), an event-driven, continuous synchroniza
 <br />
 <br />
 
-
+# Meshery Broker
+Meshery Broker is a custom Kubernetes controller that provides data streaming across independent components of Meshery whether those components are running inside or outside of the Kubernetes cluster.
+<br />
```
Comment (Chadha93) [inline README.md:39]: @pewpewnor Thanks for this, will you also add a link reference to this slide (45, 46, and 47 talks about broker) for Meshery Broker? https://docs.google.com/presentation/d/1SQMfyu5shjpGKlYONdVzOtd7UYTgLWBcgUvHMLCZ2tY/edit#slide=id.g10af036b3dd_0_0

**Comment by Ghat0tkach** [comment]
Thanks for this @pewpewnor , Please do the changes as suggested by @Chadha93

**Review by Aisuko — APPROVED** [review]
approved cc @leecalcote

File: README.md (hunk 1 above)
Comment (pewpewnor) [reply README.md:39]: Will do, I'll just add a see more about meshery broker that references to the link

Comment (pewpewnor) [reply README.md:39]: Ok, it has been done in this [commit](https://github.com/meshery/meshery-operator/pull/514/commits/e727771a940848eefb22a02b4b180e0d195d2129)

**Review by Aisuko — APPROVED** [review]
LGTM

**Comment by Aisuko** [comment]
@pewpewnor Thank you.

**Pull request merged**